lang/en.lyaml

```diff
@@ -188,11 +188,11 @@ en:
           folderOverwritePrompt: "The folder with name \"{{ folderName }}\" already exists. Overwrite?"
           errors:
             nameRequired: "The \"name\" argument is required when creating an API Sample."
-            noSamples: "Currently there are no samples available, please, try again later."
+            noSamples: "Currently there are no samples available, please try again later."
```

Comment:
mega nit: this should probably be a period and not a comma

Reply:
Good call 👌 worth fixing while I'm in here